Source/FakeItEasy.Tests/Core/ArgumentValueFormatterTests.cs

```diff
@@ -139,7 +141,25 @@ public void Should_prefer_exact_type_formatter_to_interface_formatter()
             var result = this.formatter.GetArgumentValueAsString("string value");
 
             // Assert
-            Assert.That(result, Is.EqualTo("a string"));
+            result.Should().Be("a string");
+        }
+
+        [Test]
+        public void Provided_formatters_should_have_negative_priority()

```

Comment:
Actually, it does report what's in the sequence, I forgot about that.

    new[] {"hello"}.Should().BeEmpty();

> Expected collection to be empty, but found {"hello"}.

So my remark no longer stands. We can keep the test as it is now, as far as I'm concerned.